index.html

```diff
@@ -1915,9 +1915,12 @@ <h4 id="maturity-levels">6.1.2 Maturity Levels</h4>
               corresponds to the "Last Call Working Draft" discussed in the Patent Policy.</li>
           </ul>
         </dd>
-        <dd><strong>Note</strong>: Candidate Recommendations are expected to be acceptable as Recommendations. Announcement of a
-          different next step <em class="rfc2119">should</em> include the reasons why the change in expectations comes at so late
-          a stage.</dd>
+        <dd><strong>Note</strong>: Advancing to Candidate Recommendation indicates either that
```

Comment:
The word "either" needs to be removed so this sentence makes grammatical sense.

Reply:
oops. That's a leftover from earlier phrasings. Deleting.